apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-preview.js

```diff
@@ -56,7 +55,26 @@ const TemplateSelectorPreview = ( { blocks, viewportWidth, title } ) => {
 			// Try to get the `transform` css rule from the preview container element.
 			const elStyles = window.getComputedStyle( previewContainerEl );
 			if ( elStyles && elStyles.transform ) {
-				setTransform( elStyles.transform ); // apply the same transform css rule to template title.
+				const titleElement = ref.current.querySelector( '.editor-post-title' );
+
+				// pick up scale factor from `transform` css.
+				let scale = elStyles.transform.replace( /matrix\((.+)\)$/i, '$1' ).split( ',' );
+				scale = scale && scale.length ? Number( scale[ 0 ] ) : null;
+				scale = isNaN( scale ) ? null : scale;
+
+				if ( titleElement ) {
```

Comment:
Should we group this statement with line 58 above?

Reply:
Actually, although it's completely related, technically we could apply the factor scale to the title without applying the offset to the large preview block. I've switched some lines of code, but in the end, I prefer to keep the conditions nested.